btr-api/src/btr_api/resources/submission.py

```diff
@@ -159,10 +154,9 @@ def create_register():
             # NOTE: this will be moved out of this api once lear filings are linked
             # update record in BOR (search)
             token = btr_auth.get_bearer_token()
-            entity_addresses: dict[str, dict[str, dict]] = btr_entity.get_entity_info(
-                jwt, f'{business_identifier}/addresses'
-            ).json()
-            entity['business']['addresses'] = [entity_addresses.get('registeredOffice', {}).get('deliveryAddress')]
+            address = btr_entity.get_entity_info(
+                jwt, f'{business_identifier}/addresses?addressType=deliveryAddress').json()
+            entity['business']['addresses'] = [address['deliveryAddress']]
```

Comment:
It will throw an error which is what we would want. In that case it would log the error / send the sentry message for ops to look at and continue (submission would still be successful)

This would only ever happen if there was a data issue in LEAR